### PR TITLE
Add Seasonality Detection

### PR DESCRIPTION
**Pull request by guinao**
We are excited to review your PR.

So we can do the best job, please check:

- [ ] There's a descriptive title that will make sense to other developers some time from now. 
- [ ] There's associated issues. All PR's should have issue(s) associated - unless a trivial self-evident change such as fixing a typo. You can use the format `Fixes #nnnn` in your description to cause GitHub to automatically close the issue(s) when your PR is merged.
- [ ] Your change description explains what the change does, why you chose your approach, and anything else that reviewers should know.
- [ ] You have included any necessary tests in the same PR.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/machinelearning?pullRequest=5200) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/machinelearning?pullRequest=5200) before we can accept your contribution.<br/><br/>:x: guinao [sign now](https://cla.dotnetfoundation.org/dotnet/machinelearning?pullRequest=5200)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/machinelearning?pullRequest=5200) it.</sub>

File: src/Microsoft.ML.TimeSeries/MathUtils.cs [hunk 1]
```diff
@@ -0,0 +1,121 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.ML.TimeSeries
+{
```
[the remark this answers is not in the document]
Comment (lisahua) [reply src/Microsoft.ML.TimeSeries/MathUtils.cs:10]: I will update it NLT tomorrow.

**Comment by lisahua** [comment]
Can we abandon this PR and replace it with [PR 5231](https://github.com/dotnet/machinelearning/pull/5231)?

File: src/Microsoft.ML.TimeSeries/MathUtils.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (lisahua) [reply src/Microsoft.ML.TimeSeries/MathUtils.cs:10]: Please review [PR 5231](https://github.com/dotnet/machinelearning/pull/5231) and I have asked the PR owner to abandon this PR.

**Comment by harishsk** [comment]
Closing because this has been superseded by #5231

**Pull request closed without merging**